empress/clumpr/HistogramMain.py

```diff
@@ -87,8 +87,8 @@ def transform_hist(hist, omit_zeros, xnorm, ynorm, cumulative):
 def compute_pdv(filename, tree_data, d, t, l, args):
```

Comment:
I think we might want to either put a type hint here saying that `tree_data` is of type `ReconInput` or put it in the comment.